guide/14-deep-learning/yolov3_object_detector.ipynb

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
<p>This sentence should be edited to:</p><p><em>Adapting the ResNet style residual layers has improved its accuracy while maintaining the speed advantage.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Edited. Thanks.</p>